app/views/reminder_mailer/update_user_reminder_email.html.haml

```diff
@@ -0,0 +1,10 @@
+%body
```

Comment:
By looking at the code this mail template doesn't look all to fancy :grimacing: 
Have you considered adding our logo or something?